network-api/networkapi/templates/fragments/buyersguide/about_nav.html

```diff
@@ -1,20 +1,20 @@
 {% load bg_nav_tags i18n wagtailroutablepage_tags %}
 
 {% with request.get_full_path as current_url %}
-    {% routablepageurl page 'about-why-view' as about_why_url %}
     {% routablepageurl page 'how-to-use-view' as how_to_use_url %}
+    {% routablepageurl page 'about-why-view' as about_why_url %}
     {% routablepageurl page 'methodology-view' as methodology_url %}
     {% routablepageurl page 'contact-view' as contact_url %}
     {% routablepageurl page 'press-view' as press_url %}
 
     <div class="col-lg-3">
         <nav class="vertical-nav mt-5 pt-sm-2" title="{% trans "about sections" context "Tooltip on menu items" %}">
-            <div class="multipage-link-wrapper"><a class="multipage-link about-link {% bg_active_nav current_url about_why_url %}" href="{{ about_why_url }}">{% trans "Why we made this guide" %}</a></div>
             <div class="multipage-link-wrapper"><a class="multipage-link about-link {% bg_active_nav current_url how_to_use_url %}" href="{{ how_to_use_url }}">{% trans "How to use this guide" %}</a></div>
+            <div class="multipage-link-wrapper"><a class="multipage-link about-link {% bg_active_nav current_url about_why_url %}" href="{{ about_why_url }}">{% trans "Why we made this guide" %}</a></div>
```

Comment:
Hi @robdivincenzo , thanks for updating the PR. I just wanted to double check if stakeholders are okay with "Why we made this guide" being shifted down to the list.

Reply:
Good thought, let me double-check on that w/ @kristinashu as well

Reply:
Hey @mmmavis, Kristina and Jen are both ok with the change. Thanks!

Reply:
Lovely! Thank you for letting stakeholders aware of this change.